social-network-api/src/main/java/fs/socialnetworkapi/service/AppTimezone.java

```diff
@@ -0,0 +1,14 @@
+package fs.socialnetworkapi.service;
+
+import jakarta.annotation.PostConstruct;
+import org.springframework.stereotype.Service;
+
+import java.util.TimeZone;
+
+@Service
+public class AppTimezone {
+  @PostConstruct
+  public void started() {
```

Comment:
Please, remove this class. It's not a service. It's configuration and I'm not sure that we need this.

Reply:
As I said earlier, the App is not working from my Linux OS without this solving

Reply:
In this case, it would be better to move this @PostConstruct to one of the configuration classes.